litex/soc/software/libbase/strtod.c

```diff
@@ -107,7 +107,7 @@ double strtod(const char *str, char **endptr)
   switch (*p)
     {
     case '-':
-      negative = 1; /* Fall through to increment position */
+      negative = 1; /* FALLTHRU */ /* to increment position */
```

Comment:
Are you sure you wanted this to be part of the PR?

Reply:
Yes, the text /* FALLTHRU */ is compiler magic to avoid a fall through code warning.

Reply:
OK thanks.